File: Pascal/USACO.pas
{
NAME: Harold Toomey
TASK: USACO Automated Test Template
LANG: PASCAL
}

Program USACO;
Var fin, fout: text;
    a, b: word;
Begin
    Assign(fin, 'test.in'); Reset(fin);
    Assign(fout, 'test.out'); Rewrite(fout);
    Readln(fin, a, b);
    Writeln(fout, a + b);
    Close(fout);
End.

File: Python/USACO.py
"""
NAME: Harold Toomey
TASK: USACO Automated Test Template
LANG: PYTHON2
"""

fin = open ('test.in', 'r')
fout = open ('test.out', 'w')
x,y = map(int, fin.readline().split())
sum = x + y
fout.write (str(sum) + '\n')
fout.close()
